src/systems/filecoin_vm/actor/address/address.go

```diff
@@ -55,6 +55,11 @@ func (a *Address_I) IsKeyType() bool {
 	panic("TODO")
 }
 
+func (a *Address_I) ToBytes() util.Bytes {
+	ret := make([]byte, 0)
+	return ret
```

Comment:
(see above -- would be much better if there were a conspicuous panic or util.TODO here)